src/main/webapp/data/userMapData.json

```diff
@@ -725,6 +747,7 @@
       "Rosyth School",
       "Sengkang Secondary School",
       "Serangoon Junior College",
+      "SHATEC Institutes",
```

Comment:
I think the full name is `SHATEC Institutes Pte Ltd`, which is the same as the one below.